README.md

```diff
@@ -98,22 +98,23 @@ If you want to run Clippy **only** on the given crate, use the `--no-deps` optio
 cargo clippy -p example -- --no-deps
 ```
 
-### As a rustc replacement (`clippy-driver`)
+### Projects that do not use cargo (`clippy-driver`)
 
-Clippy can also be used in projects that do not use cargo. To do so, you will need to replace
-your `rustc` compilation commands with `clippy-driver`. For example, if your project runs:
+Clippy can also be used in projects that do not use cargo. To do so, invoke `clippy-driver`
+with the same compilation commands you use for `rustc`. For example, if your project runs:
 
 ```terminal
 rustc --edition 2018 -Cpanic=abort foo.rs
 ```
 
-Then, to enable Clippy, you will need to call:
+Then, to run Clippy, you will need to run:
 
 ```terminal
 clippy-driver --edition 2018 -Cpanic=abort foo.rs
 ```
 
-Note that `rustc` will still run, i.e. it will still emit the output files it normally does.
+Note that you should still use `rustc` for compilation: it is an implementation detail whether
+`clippy-driver` runs codegen or not (and if it does, it may be different than what `rustc` outputs).
```

Comment:
How about this? (I think we can keep it high-level)

> Note: `clippy-driver` is designed for running Clippy only and should not be used as a general replacement for `rustc`. `clippy-driver` may produce artifacts that are not optimized as expected, for example.

Reply:
Done -- kept the `that` since it was already there, but let me know if you prefer the colon.